src/components/MyCoverCropList/MyCoverCropComparisonComponent.js

```diff
@@ -56,12 +56,12 @@ const lightBG = {
   minHeight: '36px',
 };
 
-const MyCoverCropComparisonComponent = (props) => {
+function MyCoverCropComparisonComponent(props) {
```

Comment:
props

Reply:
this is one of the not yet completed components yet to be worked.